package.json

```diff
@@ -23,12 +23,13 @@
     "docs": "docsify serve",
     "browser-test": "airtap --local 8080 test/browser*test.js",
     "lint": "eslint .",
-    "test": "npm run lint && tap test/*test.js test/*/*test.js && jest test/jest && npm run test-types",
-    "test-ci": "npm run lint && tap --no-check-coverage test/*test.js test/*/*test.js --coverage-report=lcovonly && npm run test-types",
-    "test-ci-pnpm": "pnpm run lint && tap --no-coverage --no-check-coverage test/*test.js test/*/*test.js && pnpm run test-types",
-    "test-ci-yarn-pnp": "yarn run lint && tap --no-check-coverage test/*test.js test/*/*test.js --coverage-report=lcovonly",
+    "test": "npm run lint && npm run transpile && tap --ts test/*test.js test/*/*test.js test/**/*.test.ts && jest test/jest && npm run test-types",
+    "test-ci": "npm run lint && npm run transpile && tap --ts --no-check-coverage test/*test.js test/*/*test.js test/**/*.test.ts --coverage-report=lcovonly && npm run test-types",
+    "test-ci-pnpm": "pnpm run lint && npm run transpile && tap --ts --no-coverage --no-check-coverage test/*test.js test/*/*test.js test/**/*.test.ts && pnpm run test-types",
+    "test-ci-yarn-pnp": "yarn run lint && npm run transpile && tap --ts --no-check-coverage test/*test.js test/*/*test.js test/**/*.test.ts --coverage-report=lcovonly",
     "test-types": "tsc && tsd && ts-node test/types/pino.ts",
-    "cov-ui": "tap --coverage-report=html test/*test.js test/*/*test.js",
+    "transpile": "sh ./test/fixtures/ts/transpile.sh",
```

Comment:
Please convert this to a cross platform script. We support development on Windows.

Reply:
Done